README.md

```diff
@@ -45,54 +45,13 @@ https://YOUR-GITHUB-USERNAME.github.io/YOUR-REPOSITORY-NAME.
 WARNING: This will override your existing `gh-pages` branch, use at your own
 risk.
 
-Options for Cargo Sphinx can be set in `Cargo.toml` under the custom section
-`package.metadata.sphinx`:
-
-* `docs-path`: string, location of the project Sphinx documentation files.
-  Default "docs".
-* `commit-message`: string, a commit message template for doc import.
-Default "(cargo-sphinx) Generate docs.".
-* `sign-commit`: bool, use GPG to sign git commits. Default false.
-* `push-remote`: string, git remote for push. Default "origin".
-* `push-branch`: string, default branch to push docs. Default "gh-pages".
-
-```toml
-[package.metadata.sphinx] 
-docs-path = "docs"
-commit-message = "(cargo-sphinx) Generate docs."
-sign-commit = false
-push-remote = "origin"
-push-branch = "gh-pages"
-```
-
 Include the `--dry-run` option to print all the commands to execute instead of
 performing the generate and upload.
 
-```
```

Comment:
This section is command output and wouldn't normally get syntax highlighting. I think the previous block is indicated `toml` by putting it after the triple ticks. The same with bash would make sense in this case, I think. The code has been moved to RestructuredText anyway. There it would be annotated like follows:

```
.. code:: bash
```

There are examples later in this patch.